### PR TITLE
Fixed broken rendering of hydra visualization paraphernalia 

### PR DESCRIPTION
**Pull request by ericrius1**
The spheres and lasers that are rendered in conjunction with the Hydra are now rendering again. This will help the user visualization where his or her hands are in the world,  as well as which direction the hydras are pointing.

### CODE REVIEW

**Comment by ZappoMan** [comment]
@ericrius1 - please add a more detailed end-user friendly description that will be included in the release notes.

File: interface/src/avatar/Avatar.cpp [hunk 1]
```diff
@@ -575,9 +575,9 @@ void Avatar::renderBody(RenderArgs* renderArgs, ViewFrustum* renderFrustum, bool
             return;
         }
 
-        if (postLighting) {
+//        if (postLighting) {
```
Comment (ZappoMan) [inline interface/src/avatar/Avatar.cpp]: dead code?

Comment (ericrius1) [inline interface/src/avatar/Avatar.cpp]: Yes - this postLighting was preventing hand rendering from being called, and it works well without that check

Comment (ericrius1) [inline interface/src/avatar/Avatar.cpp]: Never mind- it still works with it, so I added it back

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8129d0eb1ff958e86b77de51fd048a1e8965b8ae available at: https://gist.github.com/f1f0f84c3f71badd3328<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5247.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5247.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 50a7332a5a47d0a8956968a509bccf5068759185 available at: https://gist.github.com/f1f0f84c3f71badd3328<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5247.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5247.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: f49f368ec3ca565ea1709f89a90d030589a24c74 available at: https://gist.github.com/5a41a7de4ea5fa3f4136<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5247.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5247.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: a24df5c02ff2b0983d7a736385951d029816d39b available at: https://gist.github.com/f1f0f84c3f71badd3328<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5247.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5247.dmg

**Pull request merged**